config/setup_app.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cd /code/app
+
+python3 manage.py makemigrations
+
+python3 manage.py migrate
```

Comment:
The problem with writing a bash script this way is that even if the previous command fails it will throw an error but will simply proceed to execute next command in the line without caring about the result of the last executed command.

Reply:
That is why I provided the most basic commands to this. `migrations` should not cause cause errors that are not known. And in this case it would also not hurt to run any of the three when any failed.

This is just what I feel, please suggest.

Reply:
Wrapping the three commands with And operators should suffice.
```
c1 &&
c2 &&
c3
```
Or
```
python3 manage.py makemigrations
[ $? -ne 0 ] || python3 manage.py migrate
[ $? -ne 0 ] || python3 manage.py collectstatic
```